pkg/grpcserver/server.go

```diff
@@ -0,0 +1,154 @@
+package grpcserver
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"net"
+	"net/http"
+
+	grpc_middleware "github.com/grpc-ecosystem/go-grpc-middleware"
```

Comment:
this looks like too many import groups..  
usually we have 2 groups only.. stdlib and all the rest..